apps/emqx/src/emqx_schema.erl

```diff
@@ -1289,6 +1317,75 @@ fields("listener_quic_ssl_opts") ->
     server_ssl_opts_schema(#{}, false);
 fields("ssl_client_opts") ->
     client_ssl_opts_schema(#{});
+fields("ocsp") ->
+    [
+        {"enable_ocsp_stapling",
+            sc(
+                boolean(),
+                #{
+                    default => false,
+                    desc => ?DESC("server_ssl_opts_schema_enable_ocsp_stapling")
+                }
+            )},
+        {"responder_url",
+            sc(
+                binary(),
```

Comment:
There is a common typerefl reflection for URLs: `emqx_connector_http:url()`

Reply:
This returns a map of type `emqx_http_lib:uri_map()`, right?  Is there a way to use that type just to validate, but to keep this URL as a (normalized) binary?  Because we just use this by concatenating a path to this fixed URL.

Reply:
Ok. Maybe add a new reusable type for this usecase?